src/Apps/Collect/Components/ArtworkGrid/index.tsx

```diff
@@ -166,39 +166,43 @@ class Filter extends Component<Props, State> {
   }
 
   renderFilters({ user, filters, mediator, hideTopBorder }) {
-    const hasLabFeature =
-      user &&
-      user.lab_features &&
-      user.lab_features.includes("New Buy Now Flow")
-    const enableBuyNowFlow = sd.ENABLE_NEW_BUY_NOW_FLOW || hasLabFeature
+    try {
+      const hasLabFeature =
```

Comment:
Where is the error coming from here?  Not used to seeing the `try`/`catch` syntax on a function that is not declared `async` or using `await`.